modules/Administration/PasswordManager.tpl

```diff
@@ -250,8 +250,41 @@
 									{/if}
 									</table>
 
-
-
+									<table id="userResetPassId" name="userResetPassName" width="100%" border="0" cellspacing="1" cellpadding="0" class="edit view">
```

Comment:
shows even when LDAP or SAML are selected, should be hidden, unless this support them??